src/v2/view-builder/views/email/ChallengeAuthenticatorEmailView.js

```diff
@@ -11,11 +11,7 @@ const CheckYourEmailTitle = View.extend({
     'data-se': 'o-form-explain',
   },
   template: hbs`
-    {{#if email}}
-      {{i18n code="oie.email.verify.sentText" bundle="login" arguments="email" $1="<span class='strong'>$1</span>"}}
-    {{else}}
-      {{i18n code="oie.email.verify.alternate.sentText" bundle="login"}}
-    {{/if}}
+    {{i18n code="oie.email.verify.alternate.sentText" bundle="login"}}
```

Comment:
could we handle this with a secondary email instead of sending a general message?
the second option is used for user enumeration, which does not show you anything on purpose.
Can someone keep me honest here, we want to have the option for you to choose which emails to use to recover ?

Reply:
I believe the verification email is always sent to both email addresses when possible. 
You can look at this slack thread https://okta.slack.com/archives/C9AJXB5GC/p1615928217016900?thread_ts=1615923255.009200&cid=C9AJXB5GC for why we picked this option